zap/src/main/java/org/zaproxy/zap/extension/alert/AlertViewPanel.java

```diff
@@ -226,6 +227,9 @@ public void actionPerformed(ActionEvent e) {
                                 org.zaproxy.zap.model.Vulnerability v =
                                         getVulnerability((String) alertEditName.getSelectedItem());
                                 if (v != null) {
+                                    Stats.incCounter(
+                                            "stats.ui.alert.panel.vuln.selected." + v.getWascId());
```

Comment:
It would probably be better to check if the name matches a vuln when the alert is about to be raised.

Reply:
I thought about that but the names are pretty generic. I wouldn't be surprised if a user pasted or typed some of them 🤷‍♂️